vibe_code_apps_22/scripts/gh_resolve_pr76_threads.py

```
@@ -0,0 +1,116 @@
+    if proc.returncode != 0:
+        print(proc.stderr)
+        return 1
+    data = json.loads(proc.stdout)
+    nodes = data["data"]["repository"]["pullRequest"]["reviewThreads"]["nodes"]
+    unresolved = [n for n in nodes if not n["isResolved"]]
+    print(f"unresolved={len(unresolved)} total={len(nodes)}")
+    for n in unresolved:
+        cid = n["comments"]["nodes"][0]["databaseId"]
+        path = n["comments"]["nodes"][0].get("path")
+        # reply
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)